examples/ota-requestor-app/linux/main.cpp

```diff
@@ -173,7 +174,14 @@ bool HandleOptions(const char * aProgram, OptionSet * aOptions, int aIdentifier,
         gPeriodicQueryTimeoutSec = static_cast<uint32_t>(strtoul(aValue, NULL, 0));
         break;
     case kOptionRequestorCanConsent:
-        gRequestorCanConsent.SetValue(true);
+        if (strcmp(aValue, "true") == 0)
```

Comment:
What if user had a typo and aValue here was neither `true` nor `false`.  Shall we give feedback that it was an invalid argument?

Reply:
Added an error